File: dehaze-go/internal/model/sys_message_template.go
package model

// SysMessageTemplate 消息模板表
type SysMessageTemplate struct {
	BaseModel
	Code            string `gorm:"column:code;type:varchar(64);uniqueIndex:uk_code;not null;comment:模板编码" json:"code"`
	Name            string `gorm:"column:name;type:varchar(128);default:'';comment:模板名称" json:"name"`
	Type            string `gorm:"column:type;type:varchar(32);not null;comment:消息类型" json:"type"`
	TitleTemplate   string `gorm:"column:title_template;type:varchar(255);default:'';comment:标题模板" json:"titleTemplate"`
	ContentTemplate string `gorm:"column:content_template;type:text;not null;comment:正文模板" json:"contentTemplate"`
	Priority        int8   `gorm:"column:priority;type:tinyint;not null;default:2;comment:默认优先级" json:"priority"`
	Channels        string `gorm:"column:channels;type:json;comment:默认推送渠道JSON" json:"channels"`
	Variables       string `gorm:"column:variables;type:json;comment:变量定义JSON" json:"variables"`
	Status          int8   `gorm:"column:status;type:tinyint;not null;default:1;comment:状态(1:启用;0:禁用)" json:"status"`
	Deleted         int8   `gorm:"column:deleted;type:tinyint;not null;default:0;comment:逻辑删除标识" json:"deleted"`
}

func (SysMessageTemplate) TableName() string {
	return "sys_message_template"
}

File: dehaze-python/app/models/entity/sys_message_template.py
from typing import Any

from sqlalchemy import JSON, BigInteger, String, Text
from sqlalchemy.dialects import mysql as mysql_types
from sqlalchemy.orm import Mapped, mapped_column

from app.models.base import BaseModel, SoftDeleteMixin


class SysMessageTemplate(BaseModel, SoftDeleteMixin):
    __tablename__ = "sys_message_template"
    __table_args__ = {"comment": "消息模板表"}

    id: Mapped[int] = mapped_column(
        BigInteger, primary_key=True, autoincrement=True, comment="主键"
    )
    code: Mapped[str] = mapped_column(String(64), nullable=False, comment="模板编码")
    name: Mapped[str] = mapped_column(String(128), nullable=False, default="", comment="模板名称")
    type: Mapped[str] = mapped_column(String(32), nullable=False, comment="消息类型")
    title_template: Mapped[str] = mapped_column(
        String(255), nullable=False, default="", comment="标题模板"
    )
    content_template: Mapped[str] = mapped_column(Text, nullable=False, comment="正文模板")
    priority: Mapped[int] = mapped_column(
        mysql_types.TINYINT, nullable=False, default=2, comment="默认优先级"
    )
    channels: Mapped[Any | None] = mapped_column(JSON, nullable=True, comment="默认推送渠道")
    variables: Mapped[Any | None] = mapped_column(JSON, nullable=True, comment="变量定义")
    status: Mapped[int] = mapped_column(
        mysql_types.TINYINT, nullable=False, default=1, comment="状态(1:启用;0:禁用)"
    )
